Lab01_John_Zhuang.ipynb

```diff
@@ -0,0 +1,759 @@
+      "text/plain": [
+       "<Figure size 1080x720 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "n = 5\n",
+    "p = 0.85\n",
+    "\n",
+    "d_binom = stats.binom.rvs(n, p, size = 100000)\n",
+    "\n",
+    "fig, ax = plt.subplots(1, 1)\n",
+    "ax.hist(d_binom,50, density=True)\n",
+    "plt.yscale('log')\n",
+    "plt.tick_params(labelsize = 24)\n",
+    "plt.xlim([-2,8])\n",
```

Comment:
Should limit the plot and x values to 0 on the lower bound since negative values don't make any sense for this distribution